rest/config.go

```diff
@@ -719,8 +722,26 @@ func (config *ServerConfig) deprecatedConfigLoggingFallback() (warnings []base.D
 			warnings = append(warnings, func() {
 				base.Warnf(warningMsgFmt, `logging.["default"].LogFilePath`, "logging.log_file_path")
 			})
-			// Set the new LogFilePath to be the directory containing the old logfile, instead of the full path.
-			config.Logging.LogFilePath = filepath.Dir(*config.Logging.DeprecatedDefaultLog.LogFilePath)
+
```

Comment:
As discussed, let's simplify to the following:
- if config.Logging.DeprecatedDefaultLog.LogFilePath exists and is a directory, use it as the directory
- in all other cases (it's a file, or it doesn't exist) use filepath.Dir to get parent directory

Reply:
Sure Adam, included this change in the latest code.